### DISCUSSION TITLE
Helsepersonell som jobber på veiene av en annen.

### DISCUSSION THREAD

**Discussion by NEIsaksen** [NorskHelsenett/Tillitsrammeverk#69] [Q&A] [answered]
Fant ikke en lignende diskusjonstråd, så opprettet ny.

Det er ganske vanlig på legevakter, fastlegekontor andre å jobbe på veiene av andre (en ansvarlig).

Hvordan ser man på dette?
Skal innlogget stilling som jobber på veiene av andre kunne se informasjon i KJ som krever tillitsanker?
Eller er det automatikk i at om tillitsanker data ikke er overført så ser man det man ser idag.
I så fall, hvem skal de utgi seg for å være da? Seg selv eller den som de jobber på veiene av.
Mangler vi data om dette i informasjonsmodellen?

**Comment by micer2004** [comment, marked as answer]
Hei,

du tar opp et tema som er viktig å ta opp fremover, det er ikke glemt men samtidig er ikke prioritert i første omgang. Utgangspunktet var helsepersonell som logger seg i KJ-portalen hvor det ikke finnes mulighet til å logge inn på vegne av noen andre ennå.

Jeg og andre er klare over at helsepersonell kan jobbe "på vegne av" noen andre. Dette tema gjelder både autorisert og uautorisert helsepersonell (f.eks. en helsesekretær på en fastlegekontor med tilgang til EPJ som gjør spørring på vegne av fastlegen). Det som er viktigst akkurat nå er at vi får første versjon av rammeverket ut i tjenesten slik at vi kan starte diskusjoner om hvordan rammeverket skal videreutvikles. Det er mange use-case i praksis som vi må finne en passende løsning på og en del steiner som må snus slik at mer avanserte scenarioer også får støtte hos andre tjenester (f.eks. i KJ-portalen).
